addons/sourcemod/scripting/vsh/bosses/boss_horsemann.sp

```diff
@@ -91,8 +105,12 @@ methodmap CHorsemann < SaxtonHaleBase
 		char attribs[128];
 		Format(attribs, sizeof(attribs), "2 ; 2.80 ; 252 ; 0.5 ; 259 ; 1.0 ; 329 ; 0.65 ; 264 ; 0.73 ; 551 ; 1");
 		int iWeapon = this.CallFunction("CreateWeapon", 266, "tf_weapon_sword", 100, TFQual_Unusual, attribs);
+
 		if (iWeapon > MaxClients)
+		{
 			SetEntPropEnt(this.iClient, Prop_Send, "m_hActiveWeapon", iWeapon);
+			SetEntProp(iWeapon, Prop_Send, "m_nRenderMode", 0); //The boss model already has a Headtaker, so it doesn't need the original one
```

Comment:
`ability_rage_ghost.sp` have code about hiding weapon model, and reappearing after rage ends. Think that still needed if we ever want to give other bosses ghost rage?
Your render method seems better to go for `ability_rage_ghost.sp`, but that would break for HHH on rage end if we do so.